xmlsec1: Fix configure QA error caused by host lookup path

ERROR: mc:my-sdk:xmlsec1-1.2.30-r0 do_configure: QA Issue: This autoconf log indicates errors, it looked at host include and/or library paths while determining system capabilities.

It will eventually arise after the configure QA as the configure script should only look at the staging sysroot dir, not at the host.

Upstream-Status: Inappropriate [embedded specific]
Signed-off-by: Anatol Belski <anbelski@linux.microsoft.com>

--- a/configure.ac.orig   2021-01-13 14:37:42.254991177 +0000
+++ b/configure.ac        2021-01-13 14:40:56.546269330 +0000
@@ -250,8 +250,8 @@
 dnl ==========================================================================
 dnl Common installation locations
 dnl ==========================================================================
-COMMON_INCLUDE_DIR="/usr/include /usr/local/include"
-COMMON_LIB_DIR="/usr/lib /usr/lib64 /usr/local/lib"
+COMMON_INCLUDE_DIR="${STAGING_INCDIR}"
+COMMON_LIB_DIR="${STAGING_LIBDIR}"
 case $host in
     i*86-*-linux-gnu)   COMMON_LIB_DIR="$COMMON_LIB_DIR /usr/lib/i386-linux-gnu" ;;
     x86_64-*-linux-gnu) COMMON_LIB_DIR="$COMMON_LIB_DIR /usr/lib/x86_64-linux-gnu" ;;
